paf-mvp-frontend/src/utils/cookie.ts

```diff
@@ -1 +1 @@
-export const getCookieValue = (name: string): string => document.cookie.match(`(^|;)\\s*${name}\\s*=\\s*([^;]+)`)?.pop() || '';
+export const getCookieValue = (name: string): string => document.cookie.match(`(^|;)\\s*${name}\\s*=\\s*([^;]+)`)?.pop() || undefined;
```

Comment:
`: string | undefined`

Reply:
I think it would be more strictly correct to use `??` operator to allow empty strings, WDYT?

Reply:
I don't think it's possible to set an empty string in cookies.
https://stackoverflow.com/questions/8775572/setcookie-to-an-empty-value-not-working#:~:text=You%20can't%20set%20a,deleted%20from%20the%20remote%20client.

Reply:
of course, you're right ;-)
Thanks